dashboard/test/ui/features/header.feature

```diff
@@ -0,0 +1,65 @@
+@no_mobile
+Feature: Header navigation bar
+
+Scenario: Signed out user in English should see 6 header links
+  Given I am on "http://code.org/"
+  And I dismiss the language selector
+  And I wait to see "#headerlinks"
+  And I wait to see ".headerlink"
```

Comment:
And in the case where we just have static HTML, I don't think we need to check for both `#headerlinks` and `.headerlink`.